src/test/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/rollup/runner/RollupRunnerIT.kt

```diff
@@ -121,7 +123,7 @@ class RollupRunnerIT : RollupRestTestCase() {
         // Update rollup start time to run second execution
         updateRollupStartTime(rollup)
 
-        waitFor {
+        waitFor(Instant.ofEpochSecond(20)) {
```

Comment:
NP: Can remove these explicit `Instant.ofEpochSecond(20)` since default timeout of `waitFor` is now 20 seconds.